src/models/SessionReturnRecord.ts

```diff
@@ -24,7 +24,14 @@ export interface PostOfficeInfo {
 }
 
 export interface DocumentDetails {
-	documentType?: string;
+	passport?: DocumentTypeDetails[] | null;
```

Comment:
[nit] do you need ` | null` if it's an optional type?

Reply:
Sorry should've updated the PR desc better but this is the output from TxMA
![image](https://github.com/govuk-one-login/ipvreturn-api/assets/13416125/2c27de00-c7f0-4ea1-b425-ca598cc5e918)

Reply:
So like passport is there and comes through with null so that's different to having it as optional which suggests we might not get passport at all right?

Reply:
Ok yeah from what I can make out of the docs ? adds ` | undefined` which is not quite what we need 👍